user-guide/Cloud_Platform/Connecting_to_cloud/Connect_to_cloud_via_proxy.md

```diff
@@ -4,23 +4,11 @@ uid: Connect_to_cloud_via_proxy
 
 # Connecting to dataminer.services via proxy server
 
-From DataMiner CloudFeed 1.0.6, CloudGateway 2.7.0, and ArtifactDeployer 1.4.1 onwards, proxy support is available for these DxMs. This means that you can enable a proxy server so that all outgoing traffic towards the public internet will pass through that proxy server. You can use the [Proxy Cloud Pack](https://community.dataminer.services/dataminer-cloud-pack/) to install these modules on your proxy server.
+From DataMiner CloudFeed 1.0.6, CloudGateway 2.7.0, and ArtifactDeployer 1.4.1 onwards, proxy support is available for these DxMs. This means that you can enable a proxy server so that all outgoing traffic towards the public internet will pass through that proxy server.
 
-The proxy server has to support both HTTP and WebSocket traffic.
-
-To configure this:
-
-1. [Connect to dataminer.services](xref:Connect_to_dataminer_services).
+Use the [Proxy Cloud Pack](https://community.dataminer.services/dataminer-cloud-pack/) to install these modules on your proxy server.
 
-1. In the folder `C:\ProgramData\Skyline Communications\DxMs Shared\` on the DMA, configure the file *appsettings.proxy.json* as follows:
-
-   ```json
-   {
-      "ProxyOptions": {
-         "Enabled": true,
-         "Address": "<the address of the proxy server>"
-      }
-   }
-   ```
+The proxy server has to support both HTTP and WebSocket traffic.
 
-1. Restart the CloudFeed, CloudGateway, and ArtifactDeployer services for the changes to take effect.
+> [!NOTE]
+> Prior to CloudFeed 1.1.0, it was also necessary to configure the file `C:\ProgramData\Skyline Communications\DxMs Shared\appsettings.proxy.json` for this. However, CloudFeed 1.1.0 makes it possible to use CloudGateway as a proxy for the cloud connection, so that no more manual configuration is needed. If you want to connect via proxy server, you should therefore make sure you have CloudFeed 1.1.0 or higher installed.
```

Comment:
Same is valid for ArtifactDeployer, see https://github.com/SkylineCommunications/dataminer-docs/blob/6fe90cc2c7e2ce0f0417d16425772acb1021c2dc/release-notes/Cloud/DCP_change_log.md#26-april-2023---enhancement---artifactdeployer-145---artifactdeployer-no-longer-requires-direct-internet-access-id_36284